extensions/wikidata/src/org/openrefine/wikidata/schema/WbDateConstant.java

```diff
@@ -104,46 +104,51 @@ public static TimeValue parse(String datestamp)
         Date bestDate = null;
         int precision = 0; // default precision (will be overridden if successfully parsed)
         int maxLength = 0; // the maximum length parsed
+        Boolean bceFlag = Boolean.FALSE; // judge whether this is a BCE year
```

Comment:
How about using the primitive data type instead?
```suggestion
        boolean bceFlag = false; // judge whether this is a BCE year
```